### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Jacobinoo**
Remove the notice about no groups support. #13 provides this support.

### CODE REVIEW

**Comment by Jacobinoo** [comment]
#38 will be closed upon successful review

File: README.md [hunk 1]
```diff
@@ -5,9 +5,6 @@ A pure Go implementation of https://github.com/signalapp/libsignal.
 This repository is meant to be broken down into different
 APIs similar to the source repository.
 
-Only libsignal-protocol is implemented at this time,
```
Comment (RTann) [inline README.md:8]: nit: it's still the case that only `libsignal-protocol` is implemented, so maybe just removing `, and it does not support groups at this time` would be best

**Review by RTann — COMMENTED** [review]
Thank you for your contribution!

Minor nit then should be good to go.

File: README.md (hunk 1 above)
Comment (Jacobinoo) [reply README.md:8]: Yup, that’s right.

**Review by Jacobinoo — COMMENTED** [review]
Should be good now

**Review by RTann — APPROVED** [review]

**Pull request merged**